tokio-quiche/src/settings/config.rs

```diff
@@ -245,3 +245,11 @@ fn quiche_config_with_tls(
         },
     }
 }
+
+#[cfg_attr(not(feature = "rpk"), expect(dead_code))]
+fn read_file(path: &str) -> QuicResult<Vec<u8>> {
+    use anyhow::Context as _;
```

Comment:
what does the `as _` do?

Reply:
Imports it without adding it to the scope, which makes the trait's methods available without conflicting with e.g. a struct named `Context`.

See https://rust-lang.github.io/rust-clippy/master/index.html#unused_trait_names